samza-api/src/main/java/org/apache/samza/config/JavaTableConfig.java

```diff
@@ -38,6 +38,7 @@
 
   // Config keys
   public static final String TABLE_PROVIDER_FACTORY = String.format("%s.provider.factory", TABLE_ID_PREFIX);
+  public static final String DISABLE_METRICS = "disable.metrics";
```

Comment:
@weisong44 : can we use the existing config in MetricsConfig.METRICS_TIMER_DEBUG_ENABLED(metrics.timer.debug.enabled) to enable the timer metrics? Not sure we need this control on each table (I assume there will be only a few tables being used and we can turn timer metrics on/off for all of them).

The count metrics are useful since we can still infer the time from the qps, and it's cheap to calculate.

One more request in this: can we register the metrics on the container level in TableMetricsUtil.java? Usually it's too much to look at the metrics on the task level and most large applications even disable the metrics there.

Reply:
Good point, updated to use MetricsConfig.METRICS_TIMER_DEBUG_ENABLED as the global switch.